client/src/pages/ProfilePage/ProfilePage.jsx

```diff
@@ -0,0 +1,108 @@
+import { useState } from "react";
+import { useLoaderData } from "react-router-dom";
+
+import "../../scss/index.scss";
+import "./ProfilePage.scss";
+
+import ModifyInfoModale from "../../components/ModifyInfoModale/ModifyInfoModale";
+import DeleteInfoModale from "../../components/DeleteInfoModale/DeleteInfoModale";
+import DisplayCoin from "../../components/DisplayCoin/DisplayCoin";
+import ScoreDisplay from "../../components/ScoreDisplay/ScoreDisplay";
+import GameList from "../../components/GameList/GameList";
+import GameListModal from "../../components/GameListModal/GameListModal";
+
+import Avatar from "../../assets/images/avatar/Avatar-basic.svg";
+
+function ProfilePage() {
+  const gamesData = useLoaderData();
+  const [showModalModify, setShowModalModify] = useState(false);
+  const [showModalDelete, setShowModalDelete] = useState(false);
+
+  const openModalModify = () => setShowModalModify(true);
```

Comment:
you duplicate same logic, a function more generic with an inversion of state would be better